.github/workflows/cpp-packaging.yml

```diff
@@ -74,10 +97,11 @@ jobs:
   build_tools:
     name: build-tools-${{ matrix.tools_platform }}
     runs-on: ${{ matrix.os }}
+    needs: prepare_matrix
     if: ${{ github.event.inputs.downloadPublicVersion == '' && github.event.inputs.downloadPreviousRun == '' }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: ${{ fromJson(needs.prepare_matrix.outputs.matrix_build_tools_os) }}
```

Comment:
Is this just from the json for completeness? Since it has nothing to do with testing?

Reply:
I was unsure what part of this script should be made dynamic based on the expanded matrix.  I'm going to revert this part of the change for now.